src/SFA.DAS.Employer.PR.Web/Constants/ReviewPermissions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.Employer.PR.Web.Constants;
+
+public static class ReviewPermissions
```

Comment:
I am aware these exist as constants in the class PermissionModel.  I think they are better in a constant class, like you have done, so I think what you have here is fine, except call it 'ReviewPermissionsToAddRecords'.  And also make a class called 'ReviewPermissionsToRecruit' and add 'Yes', 'No', and 'Yes, employer will recruit adverts'.

Could you raise a task to conform the values in PermissionModel to these constants.  I'll raise it in SU today so it's on everyone's radar. I imagine it will be a 1 pointer, and I'm happy to pick it up when the employer web space is not in flux.